bool status
float32 x_dev
float32 y_dev
float32 z_dev
